include/mbedtls/aes.h

```diff
@@ -158,29 +162,30 @@ int mbedtls_aes_crypt_cbc( mbedtls_aes_context *ctx,
 
 #if defined(MBEDTLS_CIPHER_MODE_CFB)
 /**
- * \brief          AES-CFB128 buffer encryption/decryption.
- *
- * Note: Due to the nature of CFB you should use the same key schedule for
- * both encryption and decryption. So a context initialized with
- * mbedtls_aes_setkey_enc() for both MBEDTLS_AES_ENCRYPT and MBEDTLS_AES_DECRYPT.
- *
- * \note           Upon exit, the content of the IV is updated so that you can
- *                 call the function same function again on the following
- *                 block(s) of data and get the same result as if it was
- *                 encrypted in one call. This allows a "streaming" usage.
- *                 If on the other hand you need to retain the contents of the
- *                 IV, you should either save it manually or use the cipher
- *                 module instead.
- *
- * \param ctx      AES context
- * \param mode     MBEDTLS_AES_ENCRYPT or MBEDTLS_AES_DECRYPT
- * \param length   length of the input data
- * \param iv_off   offset in IV (updated after use)
- * \param iv       initialization vector (updated after use)
- * \param input    buffer holding the input data
- * \param output   buffer holding the output data
- *
- * \return         0 if successful
+ * \brief This function performs the operation defined in the \p mode parameter (encrypt or decrypt), on the input data buffer defined in the \p input parameter.
+ *
+ * Due to the nature of CFB, you must use the same key schedule for
+ * both encryption and decryption operations. Therefore, you must use the context initialized with
+ * mbedtls_aes_setkey_enc() for both #MBEDTLS_AES_ENCRYPT and #MBEDTLS_AES_DECRYPT.
+ *
+ * \note Upon exit, the content of the IV is updated so that you can
+ *       call the same function again on the next
+ *       block(s) of data and get the same result as if it was
+ *       encrypted in one call. This allows a "streaming" usage.
+ *       If you need to retain the contents of the
+ *       IV, you must either save it manually or use the cipher
+ *       module instead.
+ *
+ *
+ * \param ctx      The AES context to encrypt/decrypt.
+ * \param mode     The AES mode: #MBEDTLS_AES_ENCRYPT or #MBEDTLS_AES_DECRYPT
+ * \param length   The length of the input data.
+ * \param iv_off   The offset in IV (updated after use).
```

Comment:
@mjosaarinen @yanesca @mpg I don't understand what “offset in IV” refers to here, can one of you clarify?

Reply:
When used to encrypt data in multiple calls, our CFB API doesn't require you to only pass full blocks. `iv_off` is use to keep track of where in the block we are. From a user's perspective, the only thing you need to know is you should set it to `0` at the beginning of the data and then preserve its value between consecutive calls, until you want to reset and start with a fresh new IV.